tensorflow_gpu.flex.Dockerfile

```diff
@@ -60,7 +60,7 @@ ENV PYTHONPATH "${PYTHONPATH}:/workspace/src/"
 COPY --from=template_launcher /opt/google/dataflow/python_template_launcher /opt/google/dataflow/python_template_launcher
 
 # Copy files from official SDK image, including script/dependencies.
-# Note Python 3.8 is used due to the base image from nvidia
+# Note Python 3.8 is used since the above setup uses Python 3.8.
```

Comment:
just FYI, the boot entrypoint is actually the same across all beam images regardless of python versions, but it is improtant to use correct BEAM_VERSION here. and, for consistency makes sense to pick the same python version as used above..

Reply:
I meant that  /opt/apache/beam/boot  is the same in apache/beam_python3.8_sdk and apache/beam_python3.11_sdk . It's, please disregard this comment.

Reply:
I see. Got it!